build-patched-kao-kcmo-images.sh

```diff
@@ -90,5 +91,32 @@ function patch_and_push_image() {
     skopeo copy --dest-authfile ${OPENSHIFT_PULL_SECRET_PATH} --all docker://registry-proxy.engineering.redhat.com/rh-osbs/openshift-crc-${image_name}:${version}-${release} docker://quay.io/crcont/openshift-crc-${image_name}:${openshift_version}
 }
 
+function update_base_image() {
+    local brew_repo=$1
+    local base_image=$2
+
+    rhpkg clone containers/${brew_repo}
+    pushd ${brew_repo}
+    git checkout --track origin/crc-1-rhel-8
+    git checkout --track origin/private-cfergeau
+    git reset --hard origin/crc-1-rhel-8
+    sed -i "s!^FROM openshift/ose-base.*!FROM $base_image!" Dockerfile
+    git add Dockerfile
+    git commit -m "Use OpenShift ${openshift_version} base image"
+    git push origin -f HEAD:private-cfergeau
+    rhpkg container-build --scratch --target crc-1-rhel-8-candidate
+    popd
+
+    skopeo copy --dest-authfile ${OPENSHIFT_PULL_SECRET_PATH} --all docker://registry-proxy.engineering.redhat.com/rh-osbs/${brew_repo}:latest docker://quay.io/crcont/${brew_repo#crc-}:${openshift_version}
+    skopeo copy --dest-authfile ${OPENSHIFT_PULL_SECRET_PATH} --all docker://registry-proxy.engineering.redhat.com/rh-osbs/${brew_repo}:latest docker://quay.io/crcont/${brew_repo#crc-}:latest
```

Comment:
Would've been clean solution if `--additional-tag` from `skopeo copy` also support docker transport but it only support `docker-archive` only.